products-built-on-oxen/lokinet/guides/lokinet-on-a-soho-router.md

```diff
@@ -0,0 +1,325 @@
+# running lokinet client on a soho router
+
+the nicest way to run lokinet is on a soho router, it will make the defaults on the local network it serves use lokinet dns and even tunnel all traffic over lokinet in a way that it cannot leak.
+
+## requirements
+
+hardware with:
+
+* 1 wired interface 
+* 1 wireless interface
+* ability to run debian bookworm or later
+
+### network topology
+
+the network toplogy of this box will be the following:
+
+* the upstream network going wherever it goes to get internet on `eth0`
+* LAN range is 10.200.0.1/24 on `wifi0`
+* Lokinet range is 10.100.0.1/16 on `lokinet0`
+
+note the interface names for `eth0` and `wifi0` will very likely be different, these are placeholder values for the purposes of this tutorial, please replace them with the appropriate values on your system.
```

Comment:
"... likely be different, and your network may need different network ranges.  These are placeholder ..."